applications/rag/example_notebooks/rag-data-ingest-with-kubernetes-docs.ipynb

```diff
@@ -0,0 +1,286 @@
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "# initialize parameters\n",
+        "INSTANCE_CONNECTION_NAME = os.environ.get(\"CLOUDSQL_INSTANCE_CONNECTION_NAME\", \"\")\n",
+        "print(f\"Your instance connection name is: {INSTANCE_CONNECTION_NAME}\")\n",
+        "cloud_variables = INSTANCE_CONNECTION_NAME.split(\":\")\n",
+        "\n",
+        "GCP_PROJECT_ID = os.environ.get(\"GCP_PROJECT_ID\", cloud_variables[0])\n",
+        "GCP_CLOUD_SQL_REGION = os.environ.get(\"CLOUDSQL_INSTANCE_REGION\", cloud_variables[1])\n",
+        "GCP_CLOUD_SQL_INSTANCE = os.environ.get(\"CLOUDSQL_INSTANCE\", cloud_variables[2])\n",
+        "\n",
+        "DB_NAME = os.environ.get(\"INSTANCE_CONNECTION_NAME\", \"pgvector-database\")\n",
+        "VECTOR_EMBEDDINGS_TABLE_NAME = os.environ.get(\"EMBEDDINGS_TABLE_NAME\", \"rag_vector_embeddings\")\n",
+        "\n",
+        "db_username_file = open(\"/etc/secret-volume/username\", \"r\")\n",
+        "DB_USER = db_username_file.read()\n",
+        "db_username_file.close()\n",
+        "\n",
+        "db_password_file = open(\"/etc/secret-volume/password\", \"r\")\n",
+        "DB_PASS = db_password_file.read()\n",
+        "db_password_file.close()\n",
+        "\n",
+        "# Create Cloud SQL Postgres Engine\n",
+        "pg_engine = PostgresEngine.from_instance(\n",
+        "    project_id=GCP_PROJECT_ID,\n",
+        "    instance=GCP_CLOUD_SQL_INSTANCE,\n",
+        "    region=GCP_CLOUD_SQL_REGION,\n",
+        "    database=DB_NAME,\n",
+        "    user=DB_USER,\n",
+        "    password=DB_PASS,\n",
+        "    ip_type=IPTypes.PRIVATE\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "Next we'll setup some parameters for the dataset processing steps:"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "SENTENCE_TRANSFORMER_MODEL = \"intfloat/multilingual-e5-small\"  # Transformer to use for converting text chunks to vector embeddings\n",
+        "\n",
+        "# the dataset has been pre-dowloaded to the GCS bucket as part of the notebook in the cell above. Ray workers will find the dataset readily mounted.\n",
+        "SHARED_DATASET_BASE_PATH = \"/data/kubernetes-docs/\"\n",
+        "\n",
+        "BATCH_SIZE = 100\n",
+        "CHUNK_SIZE = 1000  # text chunk sizes which will be converted to vector embeddings\n",
+        "CHUNK_OVERLAP = 10\n",
+        "VECTOR_DIMENSION = 384  # Embeddings size"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Initialize Vector Store Table\n",
+        "\n",
+        "We are ready to begin. Let's first create some code for generating the vector embeddings:"
+      ]
+    },
+    {
+      "cell_type": "code",
```

Comment:
I got error `InvalidPasswordError: password authentication failed for user "rag-user"` for this cell. Any insight?

Reply:
@gongmax the password is taken from the system information, specifically from `/etc/secret-volume/password`, did you change that file?

Reply:
I didn't do any change. I just simply followed the README and this notebook